flink-core/src/main/java/org/apache/flink/api/common/functions/util/PrintSinkOutputWriter.java

```diff
@@ -33,7 +33,7 @@ public class PrintSinkOutputWriter<IN> implements Serializable, SinkWriter<IN> {
     private static final boolean STD_OUT = false;
     private static final boolean STD_ERR = true;
 
```

Comment:
```suggestion
```

Could these 2 lines be removed after refactoring the target to `isStdErr`?

Reply:
Good thinking.